scripts/ci/install-minikube.sh

```diff
@@ -6,13 +6,13 @@ OS=linux
 ARCH=amd64
 TARGET_DIR=/usr/bin
 
-MINIKUBE_VERSION="v0.25.2"
+MINIKUBE_VERSION="v0.28.2"
 MINIKUBE_URL="https://github.com/kubernetes/minikube/releases/download/$MINIKUBE_VERSION/minikube-$OS-$ARCH"
 
-KUBECTL_VERSION="v1.9.4"
-KUBECTL_URL="https://storage.googleapis.com/kubernetes-release/release/$KUBECTL_VERSION/bin/$OS/$ARCH/kubectl"
+K8S_VERSION="v1.10.0"
+KUBECTL_URL="https://storage.googleapis.com/kubernetes-release/release/$K8S_VERSION/bin/$OS/$ARCH/kubectl"
 
-WITH_CALICO=true
```

Comment:
why disabling calico ?

Reply:
Because there are issues when using Calico and Istio together.

Reply:
it's out of scope of this patch
please include this in the istio patch